config/checkstyle.xml

```diff
@@ -275,4 +280,11 @@
         </module>
         <module name="CommentsIndentation"/>
     </module>
+    <module name="RegexpMultiline">
+        <property name="format" value="(class (?s).*private static final String TAG =(?s).*TagUtility\.formatTag\((?s).*\.class\);|interface |enum )"/>
```

Comment:
Can you document and/or format this regex so people know what they need to match and what this is _supposed_ to match?

Reply:
I added a comment describing what it is looking for.